docs/overrides/main.html

```diff
@@ -2,8 +2,8 @@
 
 {% block announce %}
```

Comment:
Can we automatically load the title of the latest block post with some javascript magic?

Reply:
I can't :stuck_out_tongue: